data/deprecated.json

```diff
@@ -7,6 +7,10 @@
     "old": {"aeroway": "aerobridge"},
     "replace": {"aeroway": "jet_bridge", "highway": "corridor"}
   },
+  {
+    "old": {"aeroway": "runway", "surface": "water"},
+    "replace": {"seamark:type": "seaplane_landing_area"}
```

Comment:
Looks like we don’t have a preset for this tag yet. Specifying that preset as being applicable to an area would be a good way to ensure that the user doesn’t end up with this tag on a line if the `aeroway=runway` was tagged on a line.

Reply:
OpenSeaMap is happy to render this on a line, although the T shapes pointing to one side look unintended.

![image](https://user-images.githubusercontent.com/23022/184172382-d0dbd753-e920-477b-a19a-1c72a08a6c68.png)

Are you saying that if this is on an unclosed way we should simply not upgrade it automatically?

Reply:
Oh, I was under the impression that it’s invalid on an area. I could be wrong about that.

Reply:
(Edited my previous post, which inverted some logic confusingly.)